packages/cli/src/commands/election/current.test.ts

```diff
@@ -18,6 +19,14 @@ afterEach(async () => {
 })
 
 testWithAnvilL1('election:current cmd', async (web3: Web3) => {
+  beforeEach(async () => {
```

Comment:
this is here to reproduce the bug in the first place - otherwise tests pass just fine, but yet it doesn't work